packages/slice-machine/src/legacy/components/Navigation/Environment.tsx

```diff
@@ -110,5 +114,9 @@ export function Environment() {
     );
   }
 
-  throw useEnvironmentsError;
+  if (useEnvironmentsError !== undefined) {
+    throw useEnvironmentsError;
+  }
+
+  throw activeEnvironmentError;
```

Comment:
super weird that we throw here instead of just showing the error directly, no?

Reply:
No no, it's product decision, we want to have the global error boundary catching up that.
For this error the decision is not crash the whole app and not just display something on the side nav menu